robots.txt

```diff
@@ -0,0 +1,4 @@
+user-agent: *
+disallow: /*/3.1
+disallow: /*/3.0
+disallow: /*/2.0
```

Comment:
The current [robots.txt](http://docs.godotengine.org/robots.txt) includes the following line for the Sitemap:

```
Sitemap: https://docs.godotengine.org/sitemap.xml
```

Do we need to include that as well? The sitemap seems to reference the various translated versions of the site, and I'm not sure whether including that is needed to direct users to the docs of their particular language.

Reply:
It's to help the robots find the sitemap, just in case they wouldn't find it. Some search engines support that, others don't. But I'll add it, it doesn't hurt. Thanks!